backend/swaggerConfig.js

```
@@ -0,0 +1,267 @@
+            serviceProviderId: {
+              type: "string",
+              format: "uuid",
+              description: "ID of the provider offering this time slot",
+            },
+            isAvailable: {
+              type: "boolean",
+              description: "Availability of the time slot (true if available)",
+            },
+          },
+          required: ["id", "day", "startTime", "endTime", "serviceProviderId"],
+        },
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)